cpp/models/abm/world.cpp

```diff
@@ -100,6 +100,9 @@ void World::migration(TimePoint t, TimeSpan dt)
             std::make_pair(&go_to_icu, std::vector<LocationType>{LocationType::Hospital, LocationType::ICU}),
             std::make_pair(&go_to_quarantine, std::vector<LocationType>{LocationType::Home})};
```

Comment:
I cannot comment on the previous lines since you did not change them but I asked myself why do we set the migration rules new for every day?